Utils/github_workflow_scripts/send_slack_message.py

```diff
@@ -0,0 +1,219 @@
+
+        Returns:
+            (list): List of slack blocks representing the pack information
+    """
+    pack_name = metadata_obj.get('name')
+    version = metadata_obj.get('currentVersion')
+    support = metadata_obj.get('support')
+
+    pack_details = [
+        create_slack_section('Pack Name', pack_name),
+        create_slack_section('Support Type', support),
+        create_slack_section('Version', version),
+        {
+            "type": "divider"
+        }
+    ]
+    return pack_details
+
+
+def create_packs_segment(metadata_files: PaginatedList) -> List[dict]:
+    """Aggregate the pack information segments of the message
+
+        Args:
+            metadata_files (PaginatedList): List of File objects representing metadata files
+
+        Returns:
+            (list): List of slack blocks representing all packs information
+    """
+    all_packs = []
+    for file in metadata_files:
+        metadata_obj = get_metadata_file(file)
+        pack_segment = create_individual_pack_segment(metadata_obj)
+        all_packs += pack_segment
+    return all_packs
+
+
+def create_pull_request_segment(pr: PullRequest) -> List[dict]:
+    """Create the pull request information segment of the message
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (list): List containing a slack block-kit section entry which represents the PR info
+    """
+    assignees = ','.join([assignee.login for assignee in pr.assignees])
+    contributor = pr.user.login
+    number_of_changed_changed_files = pr.changed_files
+    labels = ','.join([label.name for label in pr.labels])
+    pr_info_segment = create_slack_fields([
+        ('Assignees', assignees),
+        ('Contributor', contributor),
+        ('Changed Files', number_of_changed_changed_files),
+        ('Labels', labels),
+    ])
+    return [pr_info_segment]
+
+
+def create_pr_title(pr: PullRequest) -> List[dict]:
+    """Create the message title
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (list): List containing a dictionary which represents the message title
+    """
+    header = [{
+        "type": "header",
+        "text": {
+            "type": "plain_text",
+            "text": f"{pr.title}",
+            "emoji": True
+        }
+    }]
+    return header
+
+
+def slack_post_message(client: WebClient, message_blocks: list, pr: PullRequest):
+    """Post a message to a slack channel
+
+        Args:
+            client (WebClient): Slack web-client object.
+            message_blocks (list): List fo blocks representing the message blocks.
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (list): List containing a dictionary which represents the message title
+    """
+    client.chat_postMessage(
+        channel="WHCL130LE",
+        # channel=SLACK_CHANNEL_TO_SEND_PR_TO,
```

Comment:
Will revert this after we will finish the PR (don't want to spam the global channel before)